iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/bicep/Decorator.java

```diff
@@ -20,10 +20,9 @@
 package org.sonar.iac.arm.tree.api.bicep;
 
 import javax.annotation.CheckForNull;
-import org.sonar.iac.arm.tree.api.ArmTree;
 import org.sonar.iac.arm.tree.api.Expression;
 
-public interface Decorator extends ArmTree {
+public interface Decorator extends HasKeyword {
```

Comment:
```suggestion
public interface Decorator extends HasKeyword, ArmTree {
```
If possible I prefer to have interfaces not relying on each other

Reply:
This would produce a code smell, not sure tbh.

Reply:
Which one?